folium/features.py

```diff
@@ -73,8 +76,9 @@ def __init__(self, url, name=None, layers=None, styles=None, format=None,
                     styles: '{{ this.styles }}',
                     format: '{{ this.format }}',
                     transparent: {{ this.transparent.__str__().lower() }},
-                    version: '{{ this.version }}',
-                    {% if this.attribution %} attribution: '{{this.attribution}}'{% endif %}
+                    version: '{{ this.version }}'
+                    {% if this.attribution %}, attribution: '{{this.attribution}}'{% endif %}
+                    {% for param in this.extra_params %}, {{param.0}}: '{{param.1}}'{% endfor %}
```

Comment:
@bielfrontera I am :+1: to all that. I am just concerned if we should be explicit there, bu naming every option, instead of looping over the `kwargs `.